src/NUnit3TestAdapterInstall/source.extension.vsixmanifest

```diff
@@ -23,7 +23,7 @@ For running NUnit 2 tests, use the NUnit 2 adapter.</Description>
         <Asset Type="UnitTestExtension" d:Source="File" Path="NUnit3.TestAdapter.dll" />
     </Assets>
     <Prerequisites>
-        <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[15.0.25904.2,16.0)" DisplayName="Visual Studio core editor" />
-        <Prerequisite Id="Microsoft.VisualStudio.Component.ManagedDesktop.Prerequisites" Version="[15.0.25904.2,16.0)" DisplayName=".NET desktop development tools" />
+        <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[15.8.27729.1,)" DisplayName="Visual Studio core editor" />
```

Comment:
Won't this stop it from installing on early versions of Visual Studio 2017 like 15.0?

Reply:
Good point.  I The minimum version has been like this for quite some time, and no complaints..... I think we did this when converting over to VS 2017.  I haven't checked with earlier versions, but I'll test it out to make sure.

Reply:
These versions are a minimum or newer. Wouldn't it be safer to leave them set where they were?

Reply:
I overlooked that, will fix.